pkg/sql/sem/tree/pretty.go

```diff
@@ -799,7 +799,11 @@ func (node *UpdateExpr) doc(p PrettyCfg) pretty.Doc {
 	if node.Tuple {
 		d = p.bracket("(", d, ")")
 	}
-	return pretty.Stack(d, pretty.Text("="), p.Doc(node.Expr))
+	e := node.Expr
+	if p.Simplify {
+		e = StripParens(e)
+	}
+	return pretty.Group(p.nestName(d, pretty.ConcatSpace(pretty.Text("="), p.Doc(e))))
```

Comment:
nestName calls Group on its outer Doc, so the call to Group here is extraneous.

Reply:
I see you fixed it. Thanks!